easybuild/easyblocks/n/netcdf.py

```diff
@@ -71,6 +71,27 @@ def configure_step(self):
 
             CMakeMake.configure_step(self)
 
+        if LooseVersion(self.version) >= LooseVersion("4.3.3"):
```

Comment:
Good question. You are probably right. The behaviour might also affected by the CMake version. (because netcdf uses some CMake provided modules to find HDF5)